framework/build.mk

```diff
@@ -116,6 +116,11 @@ pcre%.$(obj-suffix) : pcre%.cc $(PCH_DEP)
 	@$(libmesh_LIBTOOL) --tag=CXX $(LIBTOOLFLAGS) --mode=compile --quiet \
           $(libmesh_CXX) $(libmesh_CPPFLAGS) $(ADDITIONAL_CPPFLAGS) $(libmesh_CXXFLAGS) $(PCH_FLAGS) $(app_INCLUDES) $(libmesh_INCLUDE) -DHAVE_CONFIG_H -MMD -MP -MF $@.d -MT $@ -c $< -o $@
 
+%.$(obj-suffix) : %.cc $(PCH_DEP)
+	@echo "MOOSE Compiling C++ $(PCH_MODE)(in "$(METHOD)" mode) "$<"..."
```

Comment:
Note to @idaholab/moose-team: We can probably remove instances of `$(PCH_MODE)`, `$(PCH_DEP)`, etc. from the Makefiles as we are not using precompiled headers any more.